### ISSUE TITLE
INSERT INTO SELECT to load additional data step in readme errors out

### ISSUE THREAD

**Issue by ozgune** [citusdata/pgconfus-tutorial-multi-tenant#4]
When I run the INSERT INTO SELECT step in the readme, it errors out with the following:

<pre>
postgres=# INSERT INTO users SELECT * FROM tutorial.users;
ERROR:  relation "tutorial.users" does not exist
LINE 1: INSERT INTO users SELECT * FROM tutorial.users;
                                        ^
</pre>

I think this is primarily because we set up a different schema named ``tutorial`` for our PGConf SV tutorial and pre-loaded data into the tables in that schema. This way, we could load data from within AWS faster.

How big was the data set size in the ``tutorial`` schema? How easy/hard is it to scale up the sample data by 5-10x and remove that step?

**Comment by ozgune** [comment]
I continued onto the rest of the tutorial. When I ran some of the queries in queries.sql, I ran into the following error messages. I think this was because I couldn't run the INSERT INTO SELECT above.

<pre>
postgres=# INSERT INTO orders VALUES('8c69aa0d-3f13-4440-86ca-443566c1fc75',uuid_generate_v4(),'processing',12.50,'Magic_1St','Magic_1st','{"carrier": "UPS"}',now(),now(),now());
ERROR:  insert or update on table "orders_102229" violates foreign key constraint "orders_store_id_fkey_102229"
DETAIL:  Key (store_id)=(8c69aa0d-3f13-4440-86ca-443566c1fc75) is not present in table "stores_102165".
CONTEXT:  while executing command on ip-10-192-1-70.ec2.internal:5432
</pre>

**Comment by lfittl** [comment]
@ozgune Yes, I think we should rewrite this without the tutorial data being available.

There is two things we could do:

1) just drop the tutorial namespace, and take the small amount of data we have in the data directory
2) tell people to use the data generator to generate additional fake data

The downside of 2) is that people would need Ruby installed, but the benefit being that they actually have a useful amount of data.

In any case we should update `queries.sql` to reference ids that are in the data/ directory, instead of ones from the tutorial namespace.
